include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td

```diff
@@ -662,6 +671,26 @@ def TTKernel_GetInterleavedAddrGenFastOp : TTKernel_Op<"get_interleaved_addr_gen
     let results = (outs TTKernel_InterleavedAddrGenFast:$result);
 }
 
+def TTKernel_MyXOp : TTKernel_Op<"my_x"> {
+    let summary = "MyX";
+    let description = [{
+      MyX
+    }];
+
+    let arguments = (ins Optional<I8>:$noc);
+    let results = (outs Index:$x);
+}
+
+def TTKernel_MyYOp : TTKernel_Op<"my_y"> {
+    let summary = "MyY";
+    let description = [{
+      MyY
+    }];
+
+    let arguments = (ins Optional<I8>:$noc);
+    let results = (outs Index:$y);
+}
```

Comment:
Description could be improved to say something like "Return `y` co-ordinate of the NOC of the current core, if NOC ID is not provided. Else, return the y co-ordinate of the `$noc`".

CC: @nsmithtt